### PR TITLE
add truecolor output mode

### PR DESCRIPTION
**Pull request by zandrmartin**
Hi - this PR adds a true color (24-bit) color mode. Colors should be specified as `0xRRGGBB`. I don't think this breaks anything in the other output modes - I tested all the demos - but it's possible I missed something. Please let me know if any changes are needed. Thanks!

### CODE REVIEW

**Comment by nsf** [comment]
Sorry, I won't merge it. See reasoning here: https://github.com/nsf/termbox/issues/37

I really don't want people using terminals for fancy UIs. To me, even 256 color mode was a mistake. Because if we follow that road, what's next? Different font sizes and images? You know, some terminals can do that...

Join HTML cult.

**Comment by amirouche** [comment]
It's seems to me that you prefer that a large part of the community fork termbox.

File: src/termbox.h [hunk 1]
```diff
@@ -288,6 +289,9 @@ SO_IMPORT int tb_select_input_mode(int mode);
  *    This mode supports the 4th range of the 256 mode only.
  *    But you dont need to provide an offset.
  *
+ * 5. TB_OUTPUT_TRUECOLOR  => [0x000000..0xFFFFFF]
+ *    This mode supports 24-bit true color. Format is 0xRRGGBB.
+ *
  * Execute build/src/demo/output to see its impact on your terminal.
  *
  * If 'mode' is TB_OUTPUT_CURRENT, it returns the current output mode.
```
Comment (amirouche) [inline src/termbox.h]: Awesome read :heart:

**Comment by nsf** [comment]
> It's seems to me that you prefer that a large part of the community fork termbox.

That is in fact true. And I pointed it out in the issue I gave a link to. It says I'm in "maintenance only mode". Which means no new features from me and being very hesitant about merging features implemented by other people..

**Comment by SirCmpwn** [comment]
If you're concerned about the burden of ongoing maintenance, you can give write access to someone else. I would be happy to volunteer, since termbox is a dependency of some of my software. Forking is bad for everyone.

Despite what you may have come to believe, there are a great many use cases for which a TUI is suited. My software is among them.

**Pull request closed without merging**